### ISSUE TITLE
comparisons with core files

### ISSUE THREAD

**Issue by torvista** [lat9/ZCA-Bootstrap-Template#199]
It is a thankless task to keep the template's files up to date with the core files, requiring careful comparison. But its important maintenance.
This becomes much harder when the template's file may have been a bit de-loused/formatted/updated so it becomes more up-to-date than the core file to which it is being compared.
While in theory all these things can be pushed to the core, whether they get accepted or not depends on various factors/moods/complaints, the default setting seems to be if not broken, don't fix. Sometimes, maybe, depends...

My conclusion is that template files should mirror the default-template core files, warts and all, where there is no real change, just to give the comparison tools something common to hang off in both files and so we can see the few real BS changes and the core updates more clearly 
That means a file review should be done on the core file first, submitting those minor changes and waiting for yay/nay. Slow and random.
Meanwhile, when I go through a BS file to add my personal mods, I should change it to mirror default_template, even if going a bit backwards...

Thoughts??

**Comment by lat9** [comment]
I agree, but part of the reason for the reformatting/refactoring is that many of the template files required a fair amount of 'surgery' to get them to play nice with the Bootstrap framework.

I found it difficult to make those changes with the various <?php and ?> insertions as well as the 'failure to align' properly.  FWIW, I _do_ try to keep a handle on the core-file template changes (although I missed the one for #198).
